package.json

```
@@ -33,5 +33,8 @@
     "istanbul": "^0.4.1",
     "mocha": "~3.0.2",
     "walk-sync": "~0.1.2"
+  },
+  "engines": {
+    "node": "10.* || >= 12"
```

```suggestion
    "node": "10.* || 12.* || >= 14"
```

Node no longer supports 13 (since it wasn't an LTS)

---

Updated